src/ansys/dpf/post/fluid_simulation.py

```diff
@@ -218,14 +220,15 @@ def _get_result(
             Base name for the requested result.
         category:
             Type of result requested. See the :class:`ResultCategory` class.
+        native_location:
+            Native location of the result.
         location:
             Location to extract results at. Available locations are listed in
             class:`post.locations` and are: `post.locations.nodal`,
-            `post.locations.elemental`, and `post.locations.elemental_nodal`.
-            Using the default `post.locations.elemental_nodal` results in a value
-            for every node at each element. Similarly, using `post.locations.elemental`
-            gives results with one value for each element, while using `post.locations.nodal`
-            gives results with one value for each node.
+            `post.locations.elemental`, and `post.locations.faces`.
```

Comment:
Could we add post.locations.cell to comply with fluid vocabulary? 
In the same topic, I'm wondering if we should have at least some docstrings mentioning "reconstruction" (the equivalent of "averaging" in fluid vocabulary) @rafacanton. It will help if people make a search in the doc

Reply:
@cbellot000 @rafacanton the issue is there is no `gate.locations.cells`, yet `post.locations` comes from `core.locations` which comes from `gate.locations`.